tidb-server/main.go

```diff
@@ -51,7 +51,8 @@ var (
 	host            = flag.String("host", "0.0.0.0", "tidb server host")
 	port            = flag.String("P", "4000", "tidb server port")
 	statusPort      = flag.String("status", "10080", "tidb server status port")
-	lease           = flag.String("lease", "1s", "schema lease duration, very dangerous to change only if you know what you do")
+	ddlLease        = flag.String("lease", "1s", "schema lease duration, very dangerous to change only if you know what you do")
+	statsLease      = flag.String("stasLease", "1s", "stats lease duration, very dangerous to change only if you know what you do")
```

Comment:
The stats lease is not as dangerous as ddl lease.
And the default 1s is too small.